src/styles/blocks/card.css

```diff
@@ -0,0 +1,104 @@
+.card {
+  width: 200px;
+  min-height: 406px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto ;
+  
+  margin-top: 32px;
+}
+
+.card__title {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  margin: 40px 17px 0;
+  
+  color: #060b35;
+}
+
+.card__code {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 17px 0;
```

Comment:
Please add arounds paddings to card it will be smth like this:
![image](https://user-images.githubusercontent.com/54936044/224479192-07fa2640-347b-4dae-b43c-dd032dea111b.png)